go-fuzz/cover_test.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2019 Dmitry Vyukov. All rights reserved.
```

Comment:
I guess I should do s/Dmitry Vyukov/the go-fuzz project authors at this stage. Do you know if there are any quirks related to this? And if we can just rely on https://github.com/dvyukov/go-fuzz/graphs/contributors for the definition of "authors"?

Reply:
I have no earthly idea. But that’d be nice. :) Maybe ask Brad or Ian? They seem to know about such matters.